packages/design-system-tailwind-preset/src/index.test.ts

```diff
@@ -32,7 +32,9 @@ describe('Tailwind Preset', () => {
     const textColorFn = tailwindConfig.theme?.extend?.textColor as (options: {
       theme: (path: string) => unknown;
     }) => Record<string, string>;
+
     const result = textColorFn({
+      // eslint-disable-next-line jest/no-conditional-in-test -- Mocking Tailwind's theme function behavior
       theme: (path: string) => (path === 'colors' ? colors : {}),
```

Comment:
The ESLint rule jest/no-conditional-in-test is designed to prevent non-deterministic test behavior, but in these specific cases, the conditional logic is:
- Deterministic and necessary: The filter operations process data consistently based on predefined sets
- Essential for proper mocking: The theme function conditionals simulate Tailwind's actual API behavior
This approach maintains code readability and functionality while properly documenting why these specific conditionals are appropriate exceptions to the general rule.